src/compoz.ts

```diff
@@ -360,6 +407,17 @@ export class Compoz {
 			}
 		}
 
+		this.elInput.addEventListener("input", () => {
```

Comment:
I think we should not use event listener here.  Here is what will happen if we use event listener,

1. User press "t"
2. compoz call the setTimeout
3. User press "e"
4. compoz call the setTimeout
...
and so on.

_Everytime_ the input change the new setTimeout will be created.

I suggest we use `setInterval` when compoz is initialized, here is the pseudocode,

```
constructor() {
  ...
  this.historyInterval = setInterval(this.manageChanges, timeout)
}
```
and on `manageChanges` we use the body of previous setTimeout,

```
manageChanges() {
				if (this.oldChange === this.getContentHTML()) {
					return
				}
				this.historyChanges.push(this.getContentHTML())
				this.oldChange = this.getContentHTML()
				this.redoChanges = []
}
```